drivers/painter/comms/qp_comms_i2c.c

```diff
@@ -28,17 +28,15 @@ bool qp_comms_i2c_init(painter_device_t device) {
 }
 
 bool qp_comms_i2c_start(painter_device_t device) {
-    painter_driver_t *     driver       = (painter_driver_t *)device;
-    qp_comms_i2c_config_t *comms_config = (qp_comms_i2c_config_t *)driver->comms_config;
-    return i2c_start(comms_config->chip_address << 1) == I2C_STATUS_SUCCESS;
+    return true;
 }
```

Comment:
I think removal of the `qp_comms_i2c_start/stop` functions is out of scope for this PR, correct? Those functions don't exist because of the I2C drivers, but instead they exist to satisfy the `comms_start/stop` entries for QP's `painter_comms_vtable_t`. Even though they are essentially no-ops for the current I2C implementation, I could see them playing a role in threading, eventually.

Reply:
You are completely right, the implementation you've chosen is right.